contracts/LTOTokenSale.sol

```diff
@@ -244,6 +244,18 @@ contract LTOTokenSale is Ownable {
     }
   }
 
+  function withdrawFailed(address alternativeAddress) public onlyUserWithdrawalTime {
```

Comment:
We recommend to use `nonReentrant` modifier of `ReentrancyGuard` contract on this method.